docs/comparison.md

```diff
@@ -6,20 +6,20 @@ title: Comparison
 
 ## Lucide vs Feather Icons
 
-Lucide is a community-run fork of [Feather Icons](https://github.com/feathericons/feather).
+Lucide is a community-driven fork of [Feather Icons](https://github.com/feathericons/feather).
 
-It began after growing disaffection of the [Feather Icons](https://github.com/feathericons/feather) project moderation. With over 300+ open issues and over 100+ open PRs, the Feather Icons project has been abandoned and not maintained actively. This unfortunately means that hundreds of developers and designers wasted their time contributing to Feather Icons with no chance of PRs being accepted.
+The decision to create Lucide arose from growing dissatisfaction with the moderation of the Feather Icons project. With more than 300 open issues and over 100 open PRs, the Feather Icons project has been abandoned and is no longer actively maintained. Unfortunately, this means that numerous developers and designers have invested their time in contributing to Feather Icons without the possibility of their PRs being accepted.
 
-Lucide is trying to expand the icon set as much as possible while staying faithful to the original simplistic design language. We do this as a community of devs and designers.
+In an effort to expand the icon set while remaining true to the original minimalist design language, Lucide is driven by a community of developers and designers. We strive to grow together and maintain a faithful continuation of the project.
 
 ### Why should I choose Lucide over Feather Icons?
 
-- Lucide already expended the icon set by 130+ in less then a year. Lucide has over 500+ icon, feather sticks around 286 icons.
+- Lucide has already expanded the icon set by 130+ icons in less than a year. Lucide has over 500 icons, while Feather has around 286 icons.
```

Comment:
It looks like the text is outdated as well. A suggestion to update the numbers
```suggestion
- Lucide has expanded the icon set by 500+ in the last few years. Lucide has now over 1000 icons, while Feather has around 287 icons.
```

Reply:
Thanks for the update. I’ll slightly change your suggestion.

```suggestion
- Lucide has expanded its icon set by 500+ in the last few years. Lucide now has over 1000 icons, while Feather has around 287 icons.
```

Reply:
perfect!